filters/openpolicyagent/opaauthorizerequest/opaauthorizerequest_test.go

```diff
@@ -638,39 +684,42 @@ func BenchmarkAuthorizeRequest(b *testing.B) {
 						input.parsed_path = [ "allow" ]
 					}
 				`,
-			}),
-		)
+		}),
+	)
 
-		f, err := createOpaFilter(opaControlPlane)
-		assert.NoError(b, err)
+	decisionLogConsumer := newDecisionConsumer()
+	defer decisionLogConsumer.Close()
 
-		url, err := url.Parse("http://opa-authorized.test/somepath")
-		assert.NoError(b, err)
+	f, err := createOpaFilter(opaControlPlane, decisionLogConsumer, decisionLogging)
+	assert.NoError(b, err)
```

Comment:
Please use require.* (require.NoError in this case) to check preconditions and stop test early and assert.* to verify expected outcomes being tested.